run.sh

```diff
@@ -76,27 +88,38 @@ do
     if [ $DEBUG ]; then echo DEBUG: Starting loop; fi
 
     # Cleanup unused volumes
-
     if [[ $(docker version --format '{{(index .Server.Version)}}' | grep -E '^[01]\.[012345678]\.') ]]; then
       echo "=> Removing unused volumes using 'docker-cleanup-volumes.sh' script"
       /docker-cleanup-volumes.sh
     else
       echo "=> Removing unused volumes using native 'docker volume' command"
-      for volume in $(docker volume ls -qf dangling=true); do
-        echo "Deleting ${volume}"
-        docker volume rm "${volume}"
+      DANGLING_VOLUMES_IDS="`docker volume ls -qf dangling=true | xargs echo`"
```

Comment:
What was the purpose of changing this from `for volume in $(docker volume ls -qf dangling=true); do`?